src/main/plugins/org.dita.base/xsl/preprocess/maprefImpl.xsl

```diff
@@ -204,7 +204,7 @@ See the accompanying LICENSE file for applicable license.
                     <xsl:text> </xsl:text>
                     <xsl:value-of select="$target[contains(@class, ' map/map ')]/@keyscope"/>
                   </xsl:variable>
-                  <xsl:value-of select="normalize-space($keyscope)"/>
+                  <xsl:value-of select="string-join(distinct-values(tokenize($keyscope, '\s+')), ' ')"/>
```

Comment:
If the input is `"  foo    bar"`, this will output `" foo bar"`

```suggestion
                  <xsl:value-of select="string-join(distinct-values(tokenize(normalize-space($keyscope), '\s+')), ' ')"/>
```